src/actions/arcActions.ts

```diff
@@ -381,6 +382,15 @@ async function getRedemptions(avatarAddress: string, votingMachineInstance: Arc.
 
   const proposalId = proposal.proposalId;
 
+  const stakingTokenAddress = await votingMachineInstance.contract.stakingToken();
+  const StandardToken = await Arc.Utils.requireContract('StandardToken');
+  const stakingToken = await StandardToken.at(stakingTokenAddress);
+  const avatarStakingTokenBalance = Util.fromWei(await stakingToken.balanceOf(avatarAddress)).toNumber();
```

Comment:
Can you do these four lines in a single line using `votingMachineInstance.getTokenBalances`?

Reply:
fixed: https://github.com/daostack/alchemy/pull/81/commits/d1483428ad4834d4151474d0afc525e89d29d400